modules/ROOT/pages/dw-core-functions-orderby.adoc

```diff
@@ -40,7 +40,6 @@ array. Note that `orderBy($.letter)` produces the same result as
 %dw 2.0
 output application/json
 ---
-{ letter: "b", letter: "c", letter: "a" } orderBy ((value, key) -> value)
```

Comment:
Was this deleted intentionally? Should I add it to data-weave repo?

Reply:
I didn't find references in the data-weave repo about this example, maybe it was added in docs repository only. You can add it to the data-weave repo.

Reply:
After double-checking, this is OK to be deleted.